src/style.css

```
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+}
+
+.container {
+  position: relative;
+  margin-top: 20px;
+}
+
+.search-input {
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-size: 16px;
+  font-weight: 900;
+  line-height: 22px;
+  padding-left: 62px;
+  color: #3d4e61;
+  outline: none;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
```

```suggestion
  background-position: 26px 25px;
```
Ця властивість стосується не загального класу для інпутів, а саме для великого інпута. Тому варто створити і для нього клас search-input--large і прописати туди спеціифічні саме для нього стилі (як і з маленьким інпутом у тебе реалізовано)

---

підкажи ще будь ласка, коли добавив картинку як фонову, і коли переходить в стан :focus тоді картинка зникала, і я продублював параметри фонової картинки для окремого стану :focus, це нормальна поведіка що при :focus пропадає фонова картинка чи це в мене з кодом щось не так?)